src/App.tsx

```diff
@@ -53,18 +55,8 @@ export const App: React.FC = () => {
   const [isReversed, setIsReversed] = useState<boolean>(false);
   const [sortType, setSortType] = useState<SortType>(SortType.NONE);
 
-  const sortAlphabetically = () => {
-    setSortType(SortType.ALPHABET);
-    setIsReversed(false);
-  };
-
-  const reset = () => {
-    setSortType(SortType.NONE);
-    setIsReversed(false);
-  };
-
-  const sortByLength = () => {
-    setSortType(SortType.LENGTH);
+  const sortBy = (sortType: any) => {
```

Comment:
instead of `any` you should use your enum types:

`const sortBy = (sortType: SortType.NONE | SortType.ALPHABET | SortType.LENGTH) => {`